flash/image/classification/data.py

```diff
@@ -252,6 +611,114 @@ def from_csv(
         transform_kwargs: Optional[Dict] = None,
         **data_module_kwargs: Any,
     ) -> "ImageClassificationData":
+        """Load the :class:`~flash.image.classification.data.ImageClassificationData` from CSV files containing
```

Comment:
maybe explicitly mention that is parallel to `from_data_frame` just instead of data frame it is CSV

Reply:
said so, why have this function at all when you do the same with
```py
.from_data_frame(..., train_data_frame=pd.read_csv(train_csv), ...)
```

Reply:
> said so, why have this function at all when you do the same with

Just for ease of use. It will also automatically set the root to be the root of the CSV file which can be useful. I'll add something to the docs about it.